### PR TITLE
RND-96 Add helm wrapper endpoint

### PR DESCRIPTION
**Pull request by kubama**
## Description
Added helm wrapper endpoint

## Screenshots / Videos
N/A

## Checklist

- [X] My code follows the [UI Code Style](https://cloudifysource.atlassian.net/l/c/x8fq902p).
- [x] I followed the [UI Testing Guidelines](https://cloudifysource.atlassian.net/l/cp/udCVfvrx) and verified that all tests/checks have passed.
- [X] I verified if that change requires any update in the [documentation](https://cloudifysource.atlassian.net/l/c/4XKtPocy).
- [X] I added proper labels to this PR.

## Tests
Added unit test

 

## Documentation
Separate ticket

### CODE REVIEW

File: backend/handler/HelmHandler.ts [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+} from './services/BlueprintBuilder';
+
+export const renderHelmBlueprint = (renderParams: HelmRenderParams) => {
+    function createClientConfig(wrapers: Partial<Record<ClusterCredentialName, (value: any) => any>> = {}) {
+        return {
+            configuration: {
+                api_options: mapValues(
+                    renderParams.clusterCredentials,
+                    (credential, credentialName: ClusterCredentialName) =>
+                        (wrapers[credentialName] ?? identity)(
```
Comment (PatrycjaKubiczek) [inline backend/handler/HelmHandler.ts:19]: shouldn't it be with double `p` - wrappers?

File: backend/handler/HelmHandler.ts [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+import type { ClusterCredentialName, HelmRenderParams } from './HelmHandler.types';
+import {
+    createConcatCall,
+    createGetAttributeCall,
+    createGetSysCall,
+    createIntrinsicFunctionCall
+} from './services/BlueprintBuilder';
+
+export const renderHelmBlueprint = (renderParams: HelmRenderParams) => {
+    function createClientConfig(wrapers: Partial<Record<ClusterCredentialName, (value: any) => any>> = {}) {
```
Comment (PatrycjaKubiczek) [inline backend/handler/HelmHandler.ts:13]: any chances to replaces any's?

File: backend/handler/HelmHandler.ts (hunk 1 above)
Comment (kubama) [reply backend/handler/HelmHandler.ts]: It should, corrected

File: backend/handler/HelmHandler.ts (hunk 2 above)
Comment (kubama) [reply backend/handler/HelmHandler.ts]: Done

**Review by PatrycjaKubiczek — APPROVED** [review]

**Review by qooban — APPROVED** [review]
Looks good 👍🏻 

I see you moved all wrapper tests to `test/routes/blueprints`. Are you planning to also group source code in similar way (on directory structure level and endpoint naming level)?

**Comment by kubama** [comment]
> I see you moved all wrapper tests to `test/routes/blueprints`. Are you planning to also group source code in similar way (on directory structure level and endpoint naming level)?

I'll do it in the upcoming PR

**Pull request merged**